### PR TITLE
feat: Assess React Native Web

### PR DESCRIPTION
**Pull request by vadimb**

### CODE REVIEW

**Comment by vadimb** [comment]
Please don't merge yet as this will be discussed in the chapter frontend.

**Review by sasjo — CHANGES_REQUESTED** [review]
Introduce this as an assess instead of trial.

File: radar/dev/reactnative_web.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: React Native Web
+blip:
+  - date: 2021-04-30
+    ring: TRIAL
```
Comment (sasjo) [inline radar/dev/reactnative_web.yaml:4]: Change the `ring` to ASSESS.

**Review by sirenko7 — APPROVED** [review]

File: radar/dev/reactnative_web.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "React Native for Web" makes it possible to run React Native components and APIs on the web using React DOM.
+rationale:
+  Allows to have a React Native app that runs in the browser. Going forward, the trend of having a code base
+  that runs in the browser besides of JS is just about to increase with the help of WebAssembly(Flutter Web, Blazor)
+  even though RN Web isnt based on it.
+
+  Of course this is not for all types of apps but where we need to support more plaforms.
+
+  We'd like to assess this for Cloud POS and Attendant App where we need web support besides Android/iOS and
+  where maintaing a separate web interface adds more complexity for the team and time to market.
```
Comment (l0kal) [inline radar/dev/reactnative_web.yaml:15]:
```
Allows having a React Native app that runs in the browser. Going forward, the trend of having a codebase
  that runs in the browser besides JS is just about to increase with the help of WebAssembly(Flutter Web, Blazor)
  even though RN Web isn't based on it.

  Of course, this is not for all types of apps but where we need to support more platforms.

  We'd like to assess this for Cloud POS and Attendant App where we need web support besides Android/iOS and
  where maintaining a separate web interface adds more complexity for the team and time to market.
```

**Review by l0kal — CHANGES_REQUESTED** [review]
Please fix the typos and wording. I have attached a corrected text. Please advise.

**Review by sasjo — APPROVED** [review]

**Pull request merged**